Caliptra_WG_Technical_Charter.md

```diff
@@ -86,20 +112,20 @@ lists, or on calls that may be joined by any member of the public.
 Any member of the Workgroup TAC may resign by submitting a pull request to
 remove their name from the membership list document; the resignation will be
 considered effectively immediately upon submitting the pull request. The TAC may
-vote to remove a TAC member by a 2/3 vote with a majority of TAC members
-participating. A CHIPS Alliance member may remove its TAC member representative
-or replace its TAC member representative with an alternative employee
-representative at any time.
+vote to remove a TAC member by a 2/3 vote of the Voting TAC members, with 2/3
+Voting TAC members participating. A CHIPS Alliance member organization may
```

Comment:
capitalization and terminology is inconsistent. Should use the term "Voting Member" instead of including the word TAC

Reply:
sgtm

Reply:
Varun and team, I'll send a new commit with all the cleanups on language and spelling.

Awaiting for your team's ack on the proposal above

Reply:
Actually made Voting TAC Member the uniformly used term